src/test/java/com/github/daniel_sc/rocketchat/modern_client/RocketChatClientIT.java

```diff
@@ -139,6 +139,7 @@ public void testStreamMessages() throws InterruptedException {
         ChatMessage receivedMsg = msgStream.blockingFirst();
 
         assertNotNull(receivedMsg);
+        assertNotNull(receivedMsg.rid);
```

Comment:
Why not compare the actual value?
```suggestion
        assertEquals(room.rid, receivedMsg.rid);
```